src/tools/qt-gui/qml/main.qml

```diff
@@ -40,7 +40,7 @@ ApplicationWindow {
 	property bool	error: false
 	property bool	helpMode: false
 
-	property string version: "0.0.6 (beta)"
+	property string version: "0.0.7 (beta)"
```

Comment:
Well, changes to the system branch cannot be saved atm and there are still some crashes, so better play it safe.